tools/docker/Dockerfile.centos6

```diff
@@ -18,14 +18,16 @@
 FROM centos:6
 
 MAINTAINER "Apache CloudStack" <dev@cloudstack.apache.org>
-LABEL Vendor="Apache.org" License="ApacheV2" Version="4.6.0"
+LABEL Vendor="Apache.org" License="ApacheV2" Version="4.9.0-SNAPSHOT"
 
-ENV PKG_URL=http://jenkins.buildacloud.org/job/package-rhel63-master/lastSuccessfulBuild/artifact/dist/rpmbuild/RPMS/x86_64
+ENV PKG_URL=http://74.121.245.247/job/package-master-rhel63/lastSuccessfulBuild/artifact/dist/rpmbuild/RPMS/x86_64
```

Comment:
I'm fine with either ways, is the jenkins server on this IP public and can we have accounts on it?